tests/conformance.rs

```diff
@@ -2,7 +2,7 @@ use libipld_core::{ipld::Ipld, raw::RawCodec};
 use serde::{Deserialize, Serialize};
 use std::{collections::HashMap, fs::File, io::BufReader, str::FromStr};
 
-use rs_ucan::{
+use rs-ucan::{
```

Comment:
minor: I can't find @matheus23's comments now, but while the internal use of crate names remains `unclear`, `snake_case` still seems to be the convention (dash for dep of crate, _ for internal use).

Reply:
FWIW the *original* rs-ucan crate uses the `ucan` crates.io name.
Also `rs-wnfs` uses the `wnfs` crate name.
Not sure why Quinn changed the `ucan` crate name into `rs_ucan`.

Reply:
Yep. I was also going to say we shouldn't `rs` everything anyhow.

Reply:
Oh yeah, it's against the [community guidelines](https://rust-lang.github.io/api-guidelines/naming.html)

> Crate names should not use `-rs` or `-rust` as a suffix or prefix. Every crate is Rust! It serves no purpose to remind users of this constantly.

Reply:
> Not sure why Quinn changed the ucan crate name into rs_ucan.

It was me being annoyed by mistyping, and the highlighted line is me pushing a global find-and-replace

Reply:
> It was me being annoyed by mistyping, and the highlighted line is me pushing a global find-and-replace to revert that

I was referring to the fact that Quinn changed the crate name from `ucan` to `rs_ucan`, not the fact that you changed `rs_ucan` to `rs-ucan` :)